src/main/groovy/org/pih/warehouse/report/MultiLocationInventoryReportCommand.groovy

```diff
@@ -35,11 +35,11 @@ class MultiLocationInventoryReportCommand implements Validateable {
     }
 
     Boolean getIsActionRun() {
-        return actionButton?.equalsIgnoreCase("run")
+        return actionButton?.equalsIgnoreCase("run") ?: false
     }
 
     Boolean getIsActionDownload() {
-        return actionButton?.equalsIgnoreCase("download")
+        return actionButton?.equalsIgnoreCase("download") ?: false
```

Comment:
This part was throwing that `isActionRun` and `isActionDownload` cannot be null, so as a solution, I am returning false when `actionButton` is null.